src/handlers/dns/DnsStreamHandler.h

```diff
@@ -130,7 +130,7 @@ class DnsMetricsBucket final : public visor::AbstractMetricsBucket
         , _dns_topQname3("dns", "qname", {"top_qname3"}, "Top QNAMES, aggregated at a depth of three labels")
         , _dns_topNX("dns", "qname", {"top_nxdomain"}, "Top QNAMES with result code NXDOMAIN")
         , _dns_topREFUSED("dns", "qname", {"top_refused"}, "Top QNAMES with result code REFUSED")
-        , _dns_topSizedQnameResp("dns", "qname", {"top_qname_by_resp_size"}, "Top QNAMES by response size")
+        , _dns_topSizedQnameResp("dns", "qname", {"top_qname_by_resp_bytes"}, "Top QNAMES by response size in bytes")
```

Comment:
Suggest we edit the description to "Top QNAMES by response volume in bytes"

Reply:
done